force-app/main/default/classes/TestRunProcessor.cls

```diff
@@ -133,4 +138,28 @@ public with sharing class TestRunProcessor implements Schedulable {
             }
         }
     }
+
+    private void createFailureSummary(Test_Run__c testRun, List<Test_Run_Method_Result__c> failedTests) {
+        List<String> failureMessages = new List<String>();
+        for (Test_Run_Method_Result__c result : failedTests) {
+            if (result.New_Failure__c) {
+                failureMessages.add(
+                    '**  CLASS: ' +
+                    result.Name +
+                    '  **\n' +
+                    'METHOD NAME: ' +
+                    result.Method_Name__c +
+                    '\n' +
+                    'STACK TRACE: ' +
+                    result.Stack_Trace__c +
+                    '\n'
+                );
+            }
+        }
+        if (!failureMessages.isEmpty()) {
+            String failureSummary = String.join(failureMessages, '\n');
+            testRun.Failure_Summary__c = failureSummary;
```

Comment:
Maybe this should be called "New_Failure_Summary"?  Also, we might further decompose this and make it easier to manage by having the individual "Failure Summary" as a formula on the `Test_Run_Method_Result__c`.

That makes it a bit easier to make tweaks to and a bit more flexible in allow us to use it in other ways

Reply:
OR... We refactor the Email template to be a VF page and then we don't even need this field... I actually like that approach maybe better?  Would be good exposure for you to VF pages.